core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -293,11 +293,10 @@ FlowIdentifier updateSaltByName(@PathParam("workspaceId") Long workspaceId, @Pat
             @AccountId @QueryParam("accountId") String accountId);
 
     @PUT
-    @Path("{name}/pillar_config_update")
+    @Path("crn/{crn}/pillar_config_update")
```

Comment:
I think this change broke the API compatibility

Reply:
in that case please leave here the original one, and it should throw an exception and have deprecated annotation

Reply:
This was added by me in CB-6364 and is not used by anything other than this call from environment service.  It has not been pushed out and I think CRN is the proper way to go for specificity.  

And as we talked about here.
https://github.com/hortonworks/cloudbreak/pull/8640#discussion_r463534811

Reply:
yes, but unfortunately if it's in the older branch we don't have a way to ignore this and even if we merge this, all the other PR will fail with this swagger issue. there will be a procedure to remove unused APIs, but currently the only thing we can do is to keep this one, in the implementation throw an exception, and create the right one with crn. I'm not happy with this, but there is no other way around it.

Reply:
done